client/js/_author/components/assessments/question_types/movable_word_sentence/movable_word_sentence.jsx

```diff
@@ -45,7 +45,7 @@ export default class MovableWordSentence extends React.Component {
             ))
           }
           <Add
-            createChoice={() => this.props.createChoice(id)}
+            createChoice={() => this.props.createChoice()}
```

Comment:
It was not being used correctly. The id is baked into the `createChoice` method at the next level up.